packages/web/src/ssr/profile/+route.tsx

```diff
@@ -0,0 +1,3 @@
+import { makePageRoute } from 'ssr/util'
+
+export default makePageRoute('/@handle', 'Profile Page')
```

Comment:
We should probably disable this route before merging this, until we get the profile page rendering properly. Easiest way is to rename the file from `+route.tsx` probably

Reply:
Note: Chatted about this. Leaving this for now bc the page is loading properly and there should be no significant perf decrease leaving this on.